.circleci/config.yml

```diff
@@ -43,3 +48,41 @@ jobs:
           command: |
             dotnet run --project XamarinConsoleApp | tee output.txt
             grep "is True for this user" output.txt || (echo "Flag did not evaluate to expected true value" && exit 1)
+
+  build-macos:
+    macos: # We're using a Mac host because that's the only way to build for Xamarin iOS
+      xcode: "12.4.0"
+
+    steps:
+      - checkout
+
+      - run:
+          name: Install .NET/Xamarin build tools
+          command: ./.circleci/macos-install-xamarin.sh android ios
+
+      - run:
+          name: Install Android SDK
+          command: ./.circleci/macos-install-android-sdk.sh 28  # demo app currently uses Android 9.0 = API 28
+
```

Comment:
As mentioned in the previous PR, I don't know of a good way to cause the LaunchDarkly.Xamarin dependency to be updated to the latest release in the Android and iOS projects via the command line.

Reply:
Actually, never mind that - I don't think we need to worry about it. Based on inspecting the build output, it looks to me like it's already automatically picking up the latest version. The dependency version is specified as `1.*` in the Android and iOS project files, and apparently, unlike the behavior in the .NET Core project, that means the _highest_ version in that range; that would currently be 1.2.2 and that's what I'm seeing in the build. So I'm removing the line in the readme that tells people how to update it.